conformance-tests dependency leads to `Payment Required` error

A project that uses the JSpecify conformance-tests dependency started failing earlier today:

````
FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/work/checker-framework/jspecify-conformance/build.gradle' line: 96

* What went wrong:
A problem occurred evaluating root project 'jspecify-conformance'.
> Could not resolve all files for configuration ':jSpecifyConformanceTests'.
   > Could not resolve org.jspecify.conformance:conformance-tests:0.0.0-SNAPSHOT.
     Required by:
         root project 'jspecify-conformance'
      > Could not resolve org.jspecify.conformance:conformance-tests:0.0.0-SNAPSHOT.
         > Unable to load Maven meta-data from https://s01.oss.sonatype.org/content/repositories/snapshots/org/jspecify/conformance/conformance-tests/0.0.0-SNAPSHOT/maven-metadata.xml.
            > Could not GET 'https://s01.oss.sonatype.org/content/repositories/snapshots/org/jspecify/conformance/conformance-tests/0.0.0-SNAPSHOT/maven-metadata.xml'. Received status code 402 from server: Payment Required

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org./

BUILD FAILED in 8s
Error: Process completed with exit code 1.
````

A few re-runs over the day didn't fix the issue.

Maybe some budget resets tomorrow, the beginning of a new month?
Just wanted to file an issue in case somebody knows a solution or is also running into this problem.

--- a/build.gradle
+++ b/build.gradle
@@ -18,8 +18,11 @@ version = '0.0.0-SNAPSHOT'
 repositories {
     mavenLocal()
     maven {
-        name = 'ossrh-staging-api'
-        url = "https://ossrh-staging-api.central.sonatype.com/service/local/staging/deploy/maven2/"
+        name = 'Central Portal Snapshots'
+        url = 'https://central.sonatype.com/repository/maven-snapshots/'
+        content {
+            includeGroup('org.jspecify.conformance')
+        }
     }
     mavenCentral()
 }

--- a/tests/ConformanceTest-report.txt
+++ b/tests/ConformanceTest-report.txt
@@ -1,4 +1,4 @@
-# 93 pass; 24 fail; 117 total; 79.5% score
+# 99 pass; 18 fail; 117 total; 84.6% score
 PASS: Basic.java:28:test:expression-type:Object?:nullable
 PASS: Basic.java:28:test:sink-type:Object!:return
 PASS: Basic.java:28:test:cannot-convert:Object? to Object!
@@ -46,10 +46,10 @@ PASS: irrelevantannotations/notnullmarked/Other.java:Nullable exception paramete
 PASS: irrelevantannotations/notnullmarked/Other.java:NonNull exception parameter:test:irrelevant-annotation:NonNull
 PASS: irrelevantannotations/notnullmarked/Other.java:Intrinsically NonNull exception parameter cannot be assigned null:test:cannot-convert:null? to RuntimeException!
 PASS: irrelevantannotations/notnullmarked/Other.java:Nullable try-with-resources:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/notnullmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/notnullmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
 FAIL: irrelevantannotations/notnullmarked/Other.java:Nullable exception type:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/notnullmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNulll
-FAIL: irrelevantannotations/notnullmarked/Other.java: no unexpected facts
+FAIL: irrelevantannotations/notnullmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/notnullmarked/Other.java: no unexpected facts
 PASS: irrelevantannotations/nullmarked/AnnotatedTypeParameters.java:Nullable on simple type parameter on method:test:irrelevant-annotation:Nullable
 PASS: irrelevantannotations/nullmarked/AnnotatedTypeParameters.java:NonNull on simple type parameter on method:test:irrelevant-annotation:NonNull
 PASS: irrelevantannotations/nullmarked/AnnotatedTypeParameters.java:Nullable on bounded type parameter on method:test:irrelevant-annotation:Nullable
@@ -78,10 +78,10 @@ PASS: irrelevantannotations/nullmarked/Other.java:Nullable exception parameter:t
 PASS: irrelevantannotations/nullmarked/Other.java:NonNull exception parameter:test:irrelevant-annotation:NonNull
 PASS: irrelevantannotations/nullmarked/Other.java:Intrinsically NonNull exception parameter cannot be assigned null:test:cannot-convert:null? to RuntimeException!
 PASS: irrelevantannotations/nullmarked/Other.java:Nullable try-with-resources:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/nullmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/nullmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
 FAIL: irrelevantannotations/nullmarked/Other.java:Nullable exception type:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/nullmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNulll
-FAIL: irrelevantannotations/nullmarked/Other.java: no unexpected facts
+FAIL: irrelevantannotations/nullmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/nullmarked/Other.java: no unexpected facts
 PASS: irrelevantannotations/nullmarked/package-info.java: no unexpected facts
 PASS: irrelevantannotations/nullunmarked/AnnotatedTypeParameters.java:Nullable on simple type parameter on method:test:irrelevant-annotation:Nullable
 PASS: irrelevantannotations/nullunmarked/AnnotatedTypeParameters.java:NonNull on simple type parameter on method:test:irrelevant-annotation:NonNull
@@ -111,8 +111,8 @@ PASS: irrelevantannotations/nullunmarked/Other.java:Nullable exception parameter
 PASS: irrelevantannotations/nullunmarked/Other.java:NonNull exception parameter:test:irrelevant-annotation:NonNull
 PASS: irrelevantannotations/nullunmarked/Other.java:Intrinsically NonNull exception parameter cannot be assigned null:test:cannot-convert:null? to RuntimeException!
 PASS: irrelevantannotations/nullunmarked/Other.java:Nullable try-with-resources:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/nullunmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/nullunmarked/Other.java:NonNull try-with-resources:test:irrelevant-annotation:NonNull
 FAIL: irrelevantannotations/nullunmarked/Other.java:Nullable exception type:test:irrelevant-annotation:Nullable
-FAIL: irrelevantannotations/nullunmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNulll
-FAIL: irrelevantannotations/nullunmarked/Other.java: no unexpected facts
+FAIL: irrelevantannotations/nullunmarked/Other.java:NonNull exception type:test:irrelevant-annotation:NonNull
+PASS: irrelevantannotations/nullunmarked/Other.java: no unexpected facts
 PASS: irrelevantannotations/nullunmarked/package-info.java: no unexpected facts

--- a/tests/ConformanceTestOnSamples-report.txt
+++ b/tests/ConformanceTestOnSamples-report.txt
@@ -149,11 +149,11 @@ FAIL: IfCondition.java: no unexpected facts
 PASS: InferenceChoosesNullableTypeVariable.java: no unexpected facts
 FAIL: InstanceOfCheck.java:44:jspecify_nullness_mismatch
 FAIL: InstanceOfCheck.java: no unexpected facts
-FAIL: IntersectionSupertype.java:74:jspecify_nullness_mismatch
-FAIL: IntersectionSupertype.java:76:jspecify_nullness_mismatch
-FAIL: IntersectionSupertype.java:78:jspecify_nullness_mismatch
-FAIL: IntersectionSupertype.java:80:jspecify_nullness_mismatch
-FAIL: IntersectionSupertype.java:86:jspecify_nullness_mismatch
+FAIL: IntersectionSupertype.java:98:jspecify_nullness_mismatch
+FAIL: IntersectionSupertype.java:101:jspecify_nullness_mismatch
+FAIL: IntersectionSupertype.java:104:jspecify_nullness_mismatch
+FAIL: IntersectionSupertype.java:107:jspecify_nullness_mismatch
+FAIL: IntersectionSupertype.java:116:jspecify_nullness_mismatch
 FAIL: IntersectionSupertype.java: no unexpected facts
 FAIL: LocalVariable.java:44:jspecify_nullness_mismatch
 FAIL: LocalVariable.java: no unexpected facts

